components/brave_new_tab_ui/stories/default/index.tsx

```diff
@@ -54,18 +57,19 @@ export default class NewTabPage extends React.PureComponent<{}, State> {
     this.setState({ showTopSites: !this.state.showTopSites })
   }
 
-  showSettings = () => {
-    this.setState({ showSettings: true })
+  closeSettings = () => {
+    this.setState({ showSettingsMenu: false })
   }
 
-  closeSettings = () => {
-    this.setState({ showSettings: false })
+  toggleSettings = () => {
+    this.setState({ showSettingsMenu: !this.state.showSettingsMenu })
   }
 
   render () {
-    const { showSettings, showBackgroundImage, showClock, showStats, showTopSites } = this.state
+    const { showSettingsMenu, showBackgroundImage, showClock, showStats, showTopSites } = this.state
+    const { textDirection } = this.props
     return (
-      <DynamicBackground showBackgroundImage={showBackgroundImage} background={generateRandomBackgroundData.source}>
+      <DynamicBackground showBackgroundImage={showBackgroundImage} background={generateRandomBackgroundData.source} dir={textDirection}>
```

Comment:
what changes in the background when text direction changes?

Reply:
I am adding this here not as a prop but just as an HTML attribute so the page would display simulating the system preferences